app/lib/screens/mobile_registration_screen.dart

```diff
@@ -147,8 +147,16 @@ class _MobileRegistrationScreenState extends State<MobileRegistrationScreen> {
       seedConfirmationController.text = seedCheckValue;
     });
 
+    List<String> enteredWords = seedCheckValue.split(' ');
+    if(enteredWords.length != 3 || enteredWords.toSet().length != 3){
```

Comment:
more than 3 words should be allowed too
```suggestion
    if(enteredWords.length < 3 || enteredWords.toSet().length < 3){
```